site/content/integrate/apps/using-third-party-api/hello-webhooks/_index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Webhooks"
+heading: "Hello, Webhooks!"
+description: "In this example, the http app will demonstrate connect webhooks"
+weight: 90
+---
+
+This is an example of an HTTP app ([source](https://github.com/mattermost/mattermost-plugin-apps/tree/master/examples/go/hello-webhooks)), written in Go and runnable on http://localhost:8080.
+
+- It contains a `manifest.json`, declares itself an HTTP application, requests permissions, and binds itself to locations in the Mattermost user interface.
+- In its `bindings` function it declares two commands: `info` and `send`.
+- The `info` command will post an ephemeral message containing a valid `send` command.
+- The `send` command sends a webhook message to the apps plugin which in turn sends the webhook request to the `Hello, webhooks!` app
+- The hello app receives the webhook, via it's `/webhook` endpoint, and responds with an ephemeral message when the webhook is received.
+
+To install "Hello, Webhooks" on a locally-running instance of Mattermost follow these steps (go 1.16 is required):
```

Comment:
Good question.  I'd like to reorganize the Quick Start guides to so that information can be shared and linked.  For instance, any quick start guide for a specific language will share the same `prerequisites`, `bindings`, `manifest`, `icons`, `app install`, `Using the app` sections, and `Uninstalling the app` sections.

I think the following would be a much better organization.

* Quick Start Guide
  * Prerequisites
    * MM Server flags
    * Install the Apps plugin
  * Go
  * Javascript

Reply:
Added a link to the quick start prerequisites also -> [`4ca9d02` (#828)](https://github.com/mattermost/mattermost-developer-documentation/pull/828/commits/4ca9d02f33c92f1c1d1af8306f6eb3ffbac8952d)

Reply:
I like the suggested organization. Separate ticket?